examples/openshift-sample/config.yaml

```diff
@@ -0,0 +1,9 @@
+document: meta-test
+version: 1
+name: memcached
+service:
+    port: 11211
+module:
+    openshift:
```

Comment:
why isn't this part of ``examples/memcached`` and is there created new directorory for same component? Is that intentional? and is there some big differences. or this mean that ``examples/memcached/sanity1.py`` test does not work with openshift, so that just some simple one works?

Reply:
Problem is that memcached listens on port which is already used by Travis CI. Especially port 11211.

Reply:
yes, that's a problem. but still possible to integrate it, and just run some simple test instead of this what needs port mapping. but okay, we can have it in that way